README.md

```diff
@@ -1,17 +1,252 @@
 # Review Service
 
-This repository contains a simple contract to prompt user to review the current application on Android and iOS.
+This repository contains simple service that adds business logic (conditions and data tracking) around a basic review prompter for Android and iOS.
 
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE) ![Version](https://img.shields.io/nuget/v/ReviewService?style=flat-square) ![Downloads](https://img.shields.io/nuget/dt/ReviewService?style=flat-square)
+
+## Cornerstones
+
+- **Extensible**
+  - Every interface, implementation, and extension are using the generic pattern to easily allow more specific review conditions to be used for any application.
+  - Conditions are built using a builder pattern and they are based on a customizable list.
+
+- **Declarative**
+  - We are using a builder pattern for the review conditions to easily understand the behavior of the service by looking at its declaration.
+
+- **Testable**
+  - `ReviewService` is platform-agnostic, so you can easily test it or test code that uses it.
+
+## Before Getting Started
+
+Before getting started, please read the [Android](https://developer.android.com/guide/playcore/in-app-review) and [iOS](https://developer.apple.com/design/human-interface-guidelines/ratings-and-reviews) application review documentation.
 
 ## Getting Started
 
-{Instructions to quickly get started using the project: pre-requisites, packages
-to install, sample code, etc.}
+1. Add the `ReviewService` NuGet package to your projects (Windows, Android and iOS).
+   > 💡 If you need to implement more platforms or reuse code in a tests project, you can use the `ReviewService.Abstractions` NuGet package.
+
+2. Create an implementation for `IReviewSettingsSource` (which holds the review prompt settings used for prompt conditions).
+
+3. Create an instance of `ReviewService`. We'll cover dependency injection in details later on in this documentation.
+   ``` cs
+   using ReviewService;
+   using ReviewService.Abstractions;
+   
+   var reviewConditionsBuilder = ReviewConditionsBuilder.Empty()
+      .MinimumPrimaryActionsCompleted(1);
+
+   var reviewService = new ReviewService<ReviewSettings>(
+       logger: null,
+       reviewPrompter: new ReviewPrompter(logger: null),
+       reviewSettingsSource: new ReviewSettingsSource(),
+       reviewConditionsBuilder: reviewConditionsBuilder
+   )
+   ```
+
+4. Use the service.
+   1. Update the review settings based on application events (using the tracking extension methods).
```

Comment:
Don't put numbers for the sub-steps because they don't really have an order. Requesting reviews and tracking state are 2 things that need to be implemented at the same time.